_includes/scripts.html

```diff
@@ -9,6 +9,6 @@
 <script src="{{ _src | relative_url }}"></script>
 {% endfor %}{% endfor %}
 {% if layout.includes_touchpoints %}
-  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js"></script>
+  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js" integrity="sha256-bJJrp2Fye5tHOooPte/au2FqGDZwmymjr4+r4UrbeaE="></script>
```

Comment:
I think we need a `crossorigin` attribute:

```suggestion
  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js" integrity="sha256-bJJrp2Fye5tHOooPte/au2FqGDZwmymjr4+r4UrbeaE=" crossorigin="anonymous"></script>
```

Currently there's an error trying to load the script:

>Subresource Integrity: The resource 'https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js' has an integrity attribute, but the resource requires the request to be CORS enabled to check the integrity, and it is not. The resource has been blocked because the integrity cannot be enforced.